src/rpc/RPCCenter.cpp

```diff
@@ -30,41 +30,15 @@ std::unordered_set<std::string_view> const&
 handledRpcs()
 {
     static std::unordered_set<std::string_view> const kHANDLED_RPCS = {
-        "account_channels",
-        "account_currencies",
-        "account_info",
-        "account_lines",
-        "account_nfts",
-        "account_objects",
-        "account_offers",
-        "account_tx",
-        "amm_info",
-        "book_changes",
-        "book_offers",
-        "deposit_authorized",
-        "feature",
-        "gateway_balances",
-        "get_aggregate_price",
-        "ledger",
-        "ledger_data",
-        "ledger_entry",
-        "ledger_index",
-        "ledger_range",
-        "mpt_holders",
-        "nfts_by_issuer",
-        "nft_history",
-        "nft_buy_offers",
-        "nft_info",
-        "nft_sell_offers",
-        "noripple_check",
-        "ping",
-        "random",
-        "server_info",
-        "transaction_entry",
-        "tx",
-        "subscribe",
-        "unsubscribe",
-        "version",
+        "account_channels", "account_currencies", "account_info",        "account_lines",
```

Comment:
This is bad, if we need to add a new entry here (and we will), all lines will have to change (or it will be extremely ugly)